src/cli.rs

```diff
@@ -74,6 +76,53 @@ pub(crate) fn poll_for_user_input(
 		if let Some(word) = words.next() {
 			match word {
 				"help" => help(),
+				"openchannel_without_peer_addr" => {
```

Comment:
I am not sure that we need a new cli command like openchannel_without_peer_addr, but probably it is required just to allow a less restrictive parser in the ldk-sample by assuming that if the address is not specified the peer is already connected (or exist inside the ldk gossip map already)

What do you think about it?

Reply:
Yeah, that should be better.

Reply:
Regarding this, `openchannel` command should be followed by either `pubkey@host:port` or `pubkey`. 
* If it is just `pubkey`,
  * if that peer is already connected, open the fund channel
  * otherwise return error
 * if it is `pubkey@host:port`, proceed with the old behavior
 
 Does this sound good?